src/components/Donate/Donate.js

```diff
@@ -93,10 +84,18 @@ class Donate extends Component {
   }
 
   render() {
-    const { search, donations } = this.state;
+    const { search, donations, loading } = this.state;
 
     let filteredDonations = this.filterDonations(donations);
 
+    if (loading) {
+      return (
+        <div className="loading-container">
+          <ui5-busyindicator active size="Medium"></ui5-busyindicator>
+        </div>
+      )
+    } 
```

Comment:
You can move the busy indicator as a parent of the <ui5-table>:

<ui5-busyindicator>
    <ui5-table>... </ui5-table>
</ui5-busyindicator>

Reply:
Fixed